src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx

```diff
@@ -193,6 +202,46 @@ export const TradeQuoteLoaded: FC<TradeQuoteProps> = ({
     [quote?.steps],
   )
 
+  const slippage = useMemo(() => {
+    if (!quote) return
+
+    const isError =
+      userSlippageDecimal !== undefined && quote.slippageTolerancePercentage !== userSlippageDecimal
```

Comment:
Non-blocking, but can we use a consistent vernacular for both of these? i.e `slippageDecimal` or `slipperPercentage` (perhaps `percentageDecimal`).
Reading this is a bit confusing

Reply:
Also, can we use a different terminology than error here? Not being able to set custom slippage is not an error, it's part of the flow for swappers that don't support it, so this adds to the confusion

Reply:
yeah agreed, I've pushed a commit to align things a bit better 8996988ec0cb8bf1fab291908c6cb4a3938141fc

Reply:
also a88fa6bd01b8a1d81fb6baf3a448fc14adb6e09b